code/modules/clothing/clothing.dm

```diff
@@ -38,6 +40,7 @@
 	// THESE OVERRIDE THE HIDEHAIR FLAGS
 	var/dynamic_hair_suffix = ""//head > mask for head hair
 	var/dynamic_fhair_suffix = ""//mask > head for facial hair
+	var/my_clothing_icon = 'icons/mob/uniform.dmi'

```

Comment:
This doesn't need to exist, it was only for greyscaling which is already handled automatically by human ```update_icons()``` code.